homeassistant/components/alexa/handlers.py

```diff
@@ -1200,6 +1205,17 @@ async def async_api_set_mode(
             msg = f"Entity '{entity.entity_id}' does not support Mode '{mode}'"
             raise AlexaInvalidValueError(msg)
 
+    # Remote Activity
+    if instance == f"{remote.DOMAIN}.{remote.ATTR_ACTIVITY}":
```

Comment:
Seems we could have used `elif` here to avoid additional checks like we do for the other cases

Reply:
See #123853